indra/llwindow/llwindowwin32.cpp

```diff
@@ -2308,8 +2308,6 @@ LRESULT CALLBACK LLWindowWin32::mainWindowProc(HWND h_wnd, UINT u_msg, WPARAM w_
 
     LLWindowWin32* window_imp = (LLWindowWin32*)GetWindowLongPtr(h_wnd, GWLP_USERDATA);
 
-    bool debug_window_proc = false; // gDebugWindowProc || debugLoggingEnabled("Window");
```

Comment:
`debug_window_proc` had no effect because the output it supposedly wrapped was solely controlled by `LL_INFOS("Window")`. The only effect of `if (debug_window_proc)` was to skip the cached `LL_INFOS()` test if `(! debug_window_proc)`.

Reply:
Oop, sorry for the cruft here.  I put those in to trace events while the window got multithreaded, but we don't need them any more.  Would probably be better to just delete all the code that was compile time if (false)'d out.  Should improve readability and save some microseconds.